Source/MoreDotNet.Test/Extensions/Collections/EnumerableExtensions/EmptyIfNullTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace MoreDotNet.Tests.Extensions.Collections.EnumerableExtensions
+{
+    using MoreDotNet.Extensions.Collections;
+    using Xunit;
+
+    public class EmptyIfNullTests
+    {
+        [Fact]
+        public void EmtyIfNull_ShouldReturnEqualCollection()
+        {
+            int[] input = new[] { 1, 2, 3 };
+            var actual = input.EmptyIfNull();
+            Assert.Equal(input, actual);
+        }
+
+        [Fact]
+        public void EmptyIfNull_ShouldReturnEmptyIfNull()
+        {
```

Comment:
Use the following naming convention for tests:
[MethodName]_[TestSetup]_Should[ExpectedResult]
So basically:
EmtyIfNull_ShouldReturnEqualCollection
can be written as:
 EmtyIfNull_WhenGivenNonNullEnumerable_ShouldReturnEqualCollection